### ISSUE TITLE
Unary-specific rules

### ISSUE THREAD

**Issue by nordlow** [davidshepherd7/electric-operator#94]
In C-inherited languages there is a need to unary and binary specific rules for operators such as `&` and '-' which can be used as both. It would be desirable to be able configure the rules for the unary and binary operator overload individually.

**Comment by davidshepherd7** [comment]
Yeah, that would be handy! I'll have a think about it and maybe implement it next time I feel like writing some emacs lisp.

**Comment by nordlow** [comment]
Ok, thanks for now!

**Comment by davidshepherd7** [comment]
I'll keep it as an open issue to remind me.
